### PR TITLE
Enh internal messages

### PR DESCRIPTION
**Pull request by Dalmirog-zz**

### CODE REVIEW

**Comment by Dalmirog-zz** [comment]
Thanks @shaunmarx for taking the time to review :)

File: public/app.html [hunk 1]
```diff
@@ -185,13 +163,14 @@ <h3 class="panel-title">Reply
...
             <div class="panel-body">
               <div>
-                <textarea id="reply_box" class="form-control" rows="10" ng-model="currentDiscussion.reply"></textarea>
+                <textarea id="reply_box" ng-class="{'internalReplyBox' : isInternalDiscussion}" class="form-control" rows="10" ng-model="currentDiscussion.reply"></textarea>
               </div>
 
               <div style="margin-top: 10px">
-                <div style="float: left">
-                  <button class="btn btn-success" ng-click="reply(currentDiscussion)">Reply</button>
+                <div style="float: left">
```
Comment (shaunmarx) [inline public/app.html:170]: Some unnecessary white space here

File: public/tendersmash.js [hunk 2]
```diff
@@ -121,7 +121,7 @@ app.controller("mainController", function ($scope, $http, $sce, $q, $timeout, pr
   $scope.editProfile = false;
   $scope.errorManager = errorManager;
   $scope.lists = {};
-  $scope.showSummary = false;
+  $scope.showSummary = false;
```
Comment (shaunmarx) [inline public/tendersmash.js:124]: Some more whites pace over here that should be removed

File: public/tendersmash.js [hunk 3]
```diff
@@ -335,8 +335,8 @@ app.controller("mainController", function ($scope, $http, $sce, $q, $timeout, pr
       .success(function (data) {
         if (data.comments_count !== discussion.comments_count) {
           errorManager.setError("Someone else has commented on this item.");
-        } else {
-          $http.post(discussion.href + "/comments", {body: discussion.reply, skip_spam: true})
+        } else {
```
Comment (shaunmarx) [inline public/tendersmash.js:338]: Ditto here!

**Review by shaunmarx — COMMENTED** [review]
LGTM other than some white space formatting issues.

**Pull request merged**